test/isBigInt.js

```diff
@@ -0,0 +1,79 @@
+import { assert } from 'chai';
+import * as R from 'ramda';
+
+import * as RA from '../src';
+import MAX_SAFE_INTEGER from '../src/internal/polyfills/Number.MAX_SAFE_INTEGER';
+import MIN_SAFE_INTEGER from '../src/internal/polyfills/Number.MIN_SAFE_INTEGER';
+import BigInt from './shared/BigInt';
+import args from './shared/arguments';
+import Symbol from './shared/Symbol';
+
+describe('isBigInt', function () {
+  if (BigInt !== undefined) {
+    context('given a big integer value', function () {
+      specify('should return true', function () {
+        assert.isTrue(RA.isBigInt(BigInt(9007199254740991)));
+        assert.isTrue(RA.isBigInt(BigInt(-9007199254740991)));
+        assert.isTrue(RA.isBigInt(BigInt('9007199254740991')));
+        assert.isTrue(RA.isBigInt(BigInt(10)));
+        assert.isTrue(RA.isBigInt(BigInt(-10)));
+        assert.isTrue(RA.isBigInt(BigInt(0)));
+        assert.isTrue(RA.isBigInt(BigInt(-0)));
+        assert.isTrue(RA.isBigInt(BigInt(MAX_SAFE_INTEGER)));
+        assert.isTrue(RA.isBigInt(BigInt(MIN_SAFE_INTEGER)));
+        assert.isTrue(RA.isBigInt(BigInt(Number.MAX_VALUE)));
+        assert.isTrue(RA.isBigInt(BigInt('0x1fffffffffffff')));
+        assert.isTrue(
+          RA.isBigInt(
+            BigInt('0b11111111111111111111111111111111111111111111111111111')
+          )
+        );
+      });
+    });
+
+    context('given a non-big-integer value', function () {
```

Comment:
Put this function outside of the `if` statement. This block of code is not using `BigInt` symbol anywhere so it's safe to execute always.